packages/block-library/src/post-template/style.scss

```diff
@@ -11,32 +11,21 @@
 	list-style: none;
 	padding: 0;
 
-	li {
-		clear: both;
```

Comment:
This is unrelated, but couldn't find any reason for it being here.. It certainly seems like a leftover or wrong copy/paste 😄